netebpfext/sys/net_ebpf_ext_drv.c

```diff
@@ -150,20 +150,19 @@ DriverEntry(_In_ DRIVER_OBJECT* driver_object, _In_ UNICODE_STRING* registry_pat
 
     // Request NX Non-Paged Pool when available
     ExInitializeDriverRuntime(DrvRtPoolNxOptIn);
-
-    KdPrintEx((DPFLTR_IHVDRIVER_ID, DPFLTR_INFO_LEVEL, "NetEbpfExt: DriverEntry\n"));
-
     status = _net_ebpf_ext_driver_initialize_objects(driver_object, registry_path);
-
     if (!NT_SUCCESS(status)) {
         goto Exit;
     }
 
+    NET_EBPF_EXT_LOG_ENTRY();
```

Comment:
Move up to the beginning.  otherwise line 159 will result in logging an exit with no corresponding entry.

Reply:
Yep.  That's _exactly_ what I'd originally done, but subsequent tests of my variadic macros were failing.  So digging deeper I found that ```_net_ebpf_ext_driver_initialize_objects``` is where we actually register our tracelog provider (```net_ebpf_ext_trace_initiate()```).  So all logging would only show up after that.

In fact, I was curious as to why we were registering our tracelog provider that late, and not as the very first action in ```DriverEntry()```, as that would allow us to log _all_ errors.  I _did_ try to put this as a comment in this PR update, but couldn't figure out how to do that on the PR.  I was going raise this during our meeting today.